packages/menu/src/test/mwc-menu-surface.test.ts

```diff
@@ -0,0 +1,38 @@
+/**
```

Comment:
This test will not run as the runner chooses files to run via:

`packages/{packageName}/test/mwc-{packageName}.test.js`

Because of this infra limitation for `list` I wrote all the tests in the same file with different suites

Reply:
This wasn't meant to be committed. I realised it wasn't run and put them in the surface tests. I'll delete it tomorrow